src/assets/styles/mixins.less

```diff
@@ -0,0 +1,15 @@
+﻿.right-alignment(@icon-order) {
+  right: (15 + (@icon-order * 30));
+}
+
+.svg-position {
```

Comment:
The other mixing looks good, but not so sure about `svg-position`, what does it do exactly? You should name mixing in a similar way that you would name functions.